diff --git a/node_modules/@sveltejs/kit/src/core/sync/write_types/index.js b/node_modules/@sveltejs/kit/src/core/sync/write_types/index.js
index 1e4b29a..177aa9c 100644
--- a/node_modules/@sveltejs/kit/src/core/sync/write_types/index.js
+++ b/node_modules/@sveltejs/kit/src/core/sync/write_types/index.js
@@ -394,7 +394,7 @@ function process_node(node, outdir, is_page, all_pages_have_load = true) {
 				const from = proxy.modified
 					? `./proxy${replace_ext_with_js(path.basename(file_path))}`
 					: path_to_original(outdir, file_path);
-				const type = `Kit.AwaitedProperties<Awaited<ReturnType<typeof import('${from}').load>>>`;
+				const type = `Awaited<ReturnType<typeof import('${from}').load>>`;
 				return expand ? `Expand<${type}>` : type;
 			} else {
 				return fallback;
